lib/card.rb

```diff
@@ -1,8 +1,19 @@
-# Basic class to hold card attributes
+# Basic class to hold card attributes and calculate ordinal rank
 class Card
   attr_reader :suit, :value
+
   def initialize(suit, value)
     @suit = suit
     @value = value
   end
-end
\ No newline at end of file
+
+  def get_rank
+    case @value
+    when 2..10 then @value
+    when 'K' then 13
+    when 'Q' then 12
+    when 'J' then 11
+    when 'A' then 14
+    end
```

Comment:
What if we broke out those ranks into a Hash with each string pointing to the value of the card? That way, if the value is not a number between 2 and 10, we just get the rank from the Hash based on the face card's value, giving us fewer cases to deal with.